lib/test/KettleTestUtils.http.js

```diff
@@ -200,13 +202,14 @@ kettle.test.assertResponse = function (options) {
     }
 };
 
-/** Accepts a structure containing:
- *  message {String} The assertion message
- *  errorTexts {String/Array of String} Strings which are expected to appear within the text of the "message" field of the response
- *  string {String} The received response body
- *  request {Component} The http request component which has fired
- *  statusCode {Number} The expected status code (defaults to 500 if missing)
- *  plainText {Boolean} If a plain text (and not a JSON) response is expected
+/** Asserts that an error response of a particular structure has been received.
+ * @param {Object} options The structure to be checked
+ * @param {String}    options.message - The assertion message
```

Comment:
Is the additional interstitial space meant to somehow align the names?  The errorTexts option seems to mangle this.  Perhaps describing sub-options in the function description with "see above", as we've done elsewhere?

Reply:
It's nice at least to have the possibility that the structure could be machine-checked (as this syntax seems to promise) without the bulk of having to declare a fresh type.  I've added more spaced to align this properly but the vastly unpredictable length of type names is one reason I preferred the older, unsupported style of having the type name second.

Reply:
I see your point on the machine-checking and will give that more thought in my own work.

I took a moment to look into whether we can relax the type ordering.  Even though JSDoc is by no means an official standard, the conventions reflected in tools like eslint-plugin-jsdocs seem to have congealed around the style [documented here](http://usejsdoc.org/tags-param.html#names-types-and-descriptions).

The valid-jsdoc check built into ESLint and the plugin both follows this, and it's reasonable to assume that other linting tools likely follow this convention as well.  There is the option to add our own check via the eslint plugin system, but IMO we should only do that if a we choose to use a documentation-generating tool that can also work with that syntax.